pyMBE.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, SEED, temperature=None, unit_length=None, unit_charge=None, K
         self.Kw=Kw*self.units.mol**2 / (self.units.l**2)
         self.units.define(f'reduced_energy = {self.kT}')
         self.units.define(f'reduced_length = {unit_length}')
-        self.units.define(f'reduced_charge = 1*e')
+        self.units.define('reduced_charge = 1*e')
         self.setup_df()
         return
```

Comment:
Regarding the usage of `exit()` later in the code, there:
https://github.com/pyMBE-dev/pyMBE/blob/d10d17236217f71c2c13d8c7816e065c7d9e21e2/pyMBE.py#L2392-L2394

This should probably call `sys.exit(1)`. Flagged by pylint rule R1722. Right now, when a "hard check" fails, it doesn't fail hard; instead the Python script stops with exit code 0, which is interpreted as a success by the shell script. The shell script then continues its execution.

I am not familiar with the original design decision regarding module imports, so I didn't modify that line and din't add that rule to the pylint configuration file. Currently we import modules inside the pyMBE class, and we have to refer to them via `self`, e.g. `self.sys.exit(1)` instead of `sys.exit()`. I'm not really sure if there is a benefit to importing modules this way. If the reason was to defer module import until the pyMBE object is created, then this approach doesn't actually work, because import statements inside a class are not deferred (add a print statement inside the pyMBE class body to convince yourself); imports can only be deferred inside a class method or free function. If the reason was to make these modules available via the `pmb.pd` syntax, then I would recommend importing pandas in the script instead. Then one can put these import statements at the top of the pyMBE.py file.

Reply:
@jngrad I think that we can move the module imports to the top of the `pyMBE.py` file, there was no good reason of why I decided to import them inside the pyMBE class other than it was the decision I took when I started developing the library and nobody actually pointed this out before :smile:

Reply:
Alright! I moved them to the top of the file in f116cdc46b4b.